js/roi/ROIManager.js

```diff
@@ -307,23 +300,6 @@ class ROIManager {
 
     }
 
-    async findRegionWithKey(regionKey) {
-
-        const {chr, start, end} = parseRegionKey(regionKey)
-
-        for (let set of this.roiSets) {
-            const features = await set.getFeatures(chr, start, end)
-
-            for (let feature of features) {
-                if (feature.chr === chr && feature.start >= start && feature.end <= end) {
```

Comment:
I believe this line was causing the issue as it was not an exact match for start and end but we don't need this method now.